kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -499,6 +501,63 @@ object KyuubiConf {
     .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
     .createWithDefault(10099)
 
+  val FRONTEND_MYSQL_BIND_HOST: OptionalConfigEntry[String] = buildConf("frontend.mysql.bind.host")
+    .doc("Hostname or IP of the machine on which to run the MySQL frontend service.")
+    .version("1.4.0")
+    .stringConf
+    .createOptional
+
+  val FRONTEND_MYSQL_BIND_PORT: ConfigEntry[Int] = buildConf("frontend.mysql.bind.port")
+    .doc("Port of the machine on which to run the MySQL frontend service.")
+    .version("1.4.0")
+    .intConf
+    .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
+    .createWithDefault(3309)
+
+  val FRONTEND_MYSQL_NETTY_BOSS_THREADS: ConfigEntry[Int] =
+    buildConf("frontend.mysql.netty.boss.threads")
+      .doc("Thread number in the netty boss event loop of MySQL frontend service")
+      .version("1.4.0")
+      .intConf
+      .createWithDefault(1)
+
+  val FRONTEND_MYSQL_NETTY_WORKER_THREADS: ConfigEntry[Int] =
+    buildConf("frontend.mysql.netty.worker.threads")
+      .doc("Number of thread in the netty worker event loop of MySQL frontend service")
+      .version("1.4.0")
+      .intConf
+      .createWithDefault(4)
+
+  val FRONTEND_MYSQL_EXEC_POOL_SIZE: ConfigEntry[Int] =
+    buildConf("frontend.mysql.exec.pool.size")
+      .doc("Number of threads in the command execution thread pool in MySQL frontend service")
+      .version("1.4.0")
+      .intConf
+      .createWithDefault(100)
+
+  val FRONTEND_MYSQL_EXEC_WAIT_QUEUE_SIZE: ConfigEntry[Int] =
+    buildConf("frontend.mysql.exec.pool.wait.queue.size")
+      .doc("Size of the wait queue for the command execution thread pool in MySQL frontend service")
+      .version("1.4.0")
+      .intConf
+      .createWithDefault(100)
+
+  val FRONTEND_MYSQL_EXEC_KEEPALIVE_TIME: ConfigEntry[Long] =
+    buildConf("frontend.mysql.exec.pool.keepalive.time")
+      .doc("Time(ms) that an idle async thread of the command execution thread pool will wait" +
+        " for a new task to arrive before terminating in MySQL frontend service")
+      .version("1.4.0")
+      .timeConf
+      .createWithDefault(Duration.ofSeconds(60).toMillis)
```

Comment:
we can follow the thrift for all of these.

On second thought, we can make those deprecated, e.g. `FRONTEND_WORKER_KEEPALIVE_TIME`  not deprecated and use these as defaults for all FEs?

Reply:
Updated.